### DISCUSSION TITLE
Bad signature and hash for newest ISO

### DISCUSSION THREAD

**Discussion by 8bitjoe** [Security-Onion-Solutions/securityonion#11170] [2.4] [answered]
### Version

2.4.10

### Installation Method

Security Onion ISO image

### Description

other (please provide detail below)

### Installation Type

other (please provide detail below)

### Location

other (please provide detail below)

### Hardware Specs

other (please provide detail below)

### Network Traffic Collection

other (please provide detail below)

### Network Traffic Speeds

more than 10Gbps

### Status

No, one or more services are failed (please provide detail below)

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

As the title says, downloading the 2.4.10-20230821 ISO image released on 2023/08/21 i get a sha256sum 0aaa24b8e6df50dc5e44e3139cca2af605da6131f85f949ff45be66c3810112d and MD5 sum 519b06418776219b30a6d28c98c75479.
Different from the ones mentions in the docs, on top of that, GPG mentions the signature is bad.

Is this an error on my end?

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to Security Onion 2.4.

**Comment by dougburks** [comment, marked as answer]
I just downloaded and verified successfully:
```
md5sum securityonion-2.4.10-20230821.iso
353eb36f807dc947f08f79b3dcfa420e  securityonion-2.4.10-20230821.iso
```
```
sha1sum securityonion-2.4.10-20230821.iso
b25e3bedb81bbef319dc710267e6d78422f39c56  securityonion-2.4.10-20230821.iso
```
```
sha256sum securityonion-2.4.10-20230821.iso
3d369e92feb65d14e1a981e99fa223da52c92057a037c243ad6332b6b9a6d9bc  securityonion-2.4.10-20230821.iso
```
```
gpg --verify securityonion-2.4.10-20230821.iso.sig securityonion-2.4.10-20230821.iso
gpg: Signature made Mon 21 Aug 2023 09:47:50 AM EDT
gpg:                using RSA key C804A93D36BE0C733EA196447C1060B7FE507013
gpg: Good signature from "Security Onion Solutions, LLC <info@securityonionsolutions.com>" [unknown]
gpg: WARNING: This key is not certified with a trusted signature!
gpg:          There is no indication that the signature belongs to the owner.
Primary key fingerprint: C804 A93D 36BE 0C73 3EA1  9644 7C10 60B7 FE50 7013
```
Please try downloading again. If you continue to have problems, please try from a different computer or network.

**Reply by 8bitjoe** [reply]
You sir are indeed correct, something went wrong last time i downloaded, please close this issue.
And also thank you for paying so much attention to the community, we love SO <3
